src/views/modal/cart.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
```

Comment:
This relies on being in a class, so it works fine in the Gutenberg Ticket Block, but not in the classic editor. 
I created this ticket to address the issue: #133180